redi-connect-infrastructure/README.md

```diff
@@ -2,71 +2,91 @@
 
 In this project we keep all the infrastructure redi-connect needs to run. We do that writing infrastructure as code using [terraform](https://www.terraform.io/).
 
-## Prerequisites
-
-The first time one executes terraform on an azure subscription they would have to create an azure container where they can store terraform state.
+## Prerequisites & Getting started
 
+Before using using Terraform on an Azure subscription, an Azure container is required to store Terraform state.
 
 This is the list of prerequisites required for the `connect` ci/cd pipeline:
 
-1. Before you run `terraform init` - Create the backend for azure.
+1. Create the backend for Azure.
+
+   This is step is needed only the very first time _before_ you run `terraform init`. This is because when you are under a completely new Azure subscription you dont have a storage container to store the Terraform state yet.
```

Comment:
```suggestion
   This is step is needed only the very first time _before_ you run `terraform init`. This is because when you are under a completely new Azure subscription, you don't have a storage container to store the Terraform state in yet.
```